src/experiments/sync/api.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/* globals AppConstants, browser, Components, CustomizableUI, dispatcher,
+   ExtensionCommon, ExtensionAPI, Services, XPCOMUtils */
+
+"use strict";
```

Comment:
Just a comment, nothing to take action on; I die a little on the inside every time I see it

Reply:
😂 just cargo culting the [existing style](https://searchfox.org/mozilla-central/search?q=strict&case=false&regexp=false&path=toolkit%2Fcomponents%2Fextensions%2Fparent), and now that I look, I think I copypasta'd the eslint globals incorrectly anyway. I can get rid of this

Reply:
Ah, on closer examination, there is a logical reason to use that directive: Mozilla code doesn't use ES6 modules, so strict mode isn't applied by default :-\